graphql/data/marathon/groups/server.js

```diff
@@ -0,0 +1,40 @@
+import DataLoader from 'dataloader';
+
+import config from '../../../config';
+import { fetchWithAuth } from '../../../utils/fetch';
+
+export default class GroupsServerConnector {
+  constructor({ authToken }) {
+    const query = {
+      embed: [
+        'group.groups',
+        'group.apps',
+        'group.pods',
+        'group.apps.deployments',
+        'group.apps.counts',
+        'group.apps.tasks',
+        'group.apps.taskStats',
+        'group.apps.lastTaskFailure'
+      ]
+    };
+
+    const getGroup = fetchWithAuth(authToken, {
+      baseURI: config.endpoints.marathon.groups,
+      query
+    });
+
+    this.groupsLoader = new DataLoader(getGroup, { batch: false });
+  }
+
+  get(groupId = '/') {
```

Comment:
having '/' by default and then clearing the value right at the beginning of the function looks weird,
but then most probably you can either call it either with `/` or without any arguments, right?